hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/key/OMOpenKeysDeleteRequest.java

```diff
@@ -52,7 +54,7 @@
 public class OMOpenKeysDeleteRequest extends OMKeyRequest {
 
   private static final Logger LOG =
-          LoggerFactory.getLogger(OMOpenKeysDeleteRequest.class);
+      LoggerFactory.getLogger(OMOpenKeysDeleteRequest.class);
 
   public OMOpenKeysDeleteRequest(OMRequest omRequest) {
```

Comment:
Could you please tell me the plan to invoke the request type `new OMOpenKeysDeleteRequest()`. Presently, thsi constructor is not used anywhere.

Reply:
It will be used in HDDS-4123. For example:

https://github.com/errose28/hadoop-ozone/blob/5a692520a41f5495aef8939a2afb95c120f9d149/hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OpenKeyCleanupService.java#L165-L174